tests/integration_tests/performance/test_network_tcp_throughput.py

```diff
@@ -0,0 +1,396 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests the network throughput overhead added by Firecracker."""
+
+import itertools
+import json
+import logging
+import time
+import concurrent.futures
+import re
+import pytest
+from conftest import _test_images_s3_bucket
+from framework.artifacts import ArtifactCollection, ArtifactSet
+from framework.matrix import TestMatrix, TestContext
+from framework.builder import MicrovmBuilder
+from framework.statistics import core, consumer, producer, criteria, \
+    types, function
+from framework.utils import CpuMap, ProcessCpuAffinity, CmdBuilder, run_cmd, \
+    eager_map
+import host_tools.network as net_tools
+
+CONFIG = {
+    "time": 2,  # seconds
+    "omit": 1,
+    "modes": {
+        "g2h": [""],
+        "h2g": ["-R"],
+        "bd": ["", "-R"]
+    },
+    "protocols": [
+        {
+            "name": "tcp",
+            "omit": 1,  # seconds
+            # Used as KBytes - congestion window.
+            "window_size": [10, 64, 128, 256],
```

Comment:
I think you have too many options for window size. Also you should leave it to run with default params, including packet length.

Reply:
Reduced the options for window size. PTAL!